cmd/server/assets/realmadmin/_stats_keyserver.html

```diff
@@ -308,18 +338,26 @@ <h5 class="modal-title">Key server stats</h5>
     return dataTable;
   }
 
+  
   let onsetChart;
   let onsetOptions;
   let onsetData;
+  let onsetVAxisMax;
   function drawOnsetToUploadChart(data) {
     let $div = $('#keyserver_onset_chart_div');
+    onsetVAxisMax = 1;
 
     if (!data.length) {
       $div.find('p').text('No data yet.');
       return;
     }
     stats = data;
 
+    // calculate the vaxis max.
+    for (i = 0; i < stats.length - smoothing; i++) {
+      getOnsetDataTable(i);
```

Comment:
ah now 'getOnsetDataTable' creates a bunch of new google.visualization.DataTable() to be immediately thrown away (to populate the global variable). This is getting messy.

If we want to do this maybe we should cache the results so we don't need to rebuild these as we slide the slider.

Reply:
added caches - redraws are faster now!